libandroid-navigation-ui/src/main/res/values/attrs.xml

```diff
@@ -23,6 +23,9 @@
 
     <attr name="upcomingManeuverArrowColor" format="color"/>
     <attr name="upcomingManeuverArrowBorderColor" format="color"/>
+
+    <!-- Route line cap -->
+    <attr name="routeLineCap" format="string"/>
```

Comment:
Wondering how we could be more strict here 🤔 Accepting a string could be problematic at runtime if the value passed in is not one of `"butt"`, `"round"` or `"square"` 👀 https://docs.mapbox.com/mapbox-gl-js/style-spec/#layout-line-line-cap If we're only going to allow `"round"` and `"square"`, what about using a `boolean` `attr` instead? Does that make sense?